### PR TITLE
Link repo pr

### PR DESCRIPTION
**Pull request by borek2**
Made all the repos clickable and pull requests in peer reviewers

### CODE REVIEW

File: src/aggregators/ReviewsOnYourPrsAggregator.js [hunk 1]
```diff
@@ -60,10 +60,22 @@ function ReviewOnYourPrsAggregator(userName, platform) {
...
+            }));
+        });
+        return RSVP.all(promises);
+    }
+    
     function transformToGraphObject(userSessions) {
         return userSessions.map(function (us) {
             return {
                 "pr": us.pull_request.pull_request_number,
+                "pr_url": us.pull_request.prInfo.url,
```
Comment (DaanvanderValk) [inline src/aggregators/ReviewsOnYourPrsAggregator.js]: Is this the GitHub website link, or the corresponding API address?

**Comment by agudek** [comment]
The force layout tooltip link to the repo does not have the correct url yet

**Comment by agudek** [comment]
Also, could the repo_url also be placed inside the pr nodes?

**Comment by agudek** [comment]
Fixed!

**Pull request merged**